.ci/scripts/test_wheel_package_qnn.sh

```diff
@@ -139,8 +139,30 @@ run_core_tests () {
   echo "=== [$LABEL] Installing wheel & deps ==="
   "$PIPBIN" install --upgrade pip
   "$PIPBIN" install "$WHEEL_FILE"
-  "$PIPBIN" install torch=="2.9.0.dev20250906" --index-url "https://download.pytorch.org/whl/nightly/cpu"
-  "$PIPBIN" install --pre torchao --index-url "https://download.pytorch.org/whl/nightly/cpu"
+  TORCH_VERSION=$(
+  "$PYBIN" - <<'PY'
+import re, pathlib
+txt = pathlib.Path("install_requirements.py").read_text()
+print(re.search(r'TORCH_VERSION\s*=\s*["\']([^"\']+)["\']', txt).group(1))
+PY
+)
+
+  NIGHTLY_VERSION=$(
+  "$PYBIN" - <<'PY'
+import re, pathlib
+txt = pathlib.Path("install_requirements.py").read_text()
+print(re.search(r'NIGHTLY_VERSION\s*=\s*["\']([^"\']+)["\']', txt).group(1))
+PY
+)
+  echo "=== [$LABEL] Install torch==${TORCH_VERSION}.${NIGHTLY_VERSION} ==="
+
+  # Install torchao based on the pinned PyTorch version
+  "$PIPBIN" install torch=="${TORCH_VERSION}.${NIGHTLY_VERSION}" --index-url "https://download.pytorch.org/whl/nightly/cpu"
+
+  # Install torchao based on the pinned commit from third-party/ao submodule
+  pushd "$REPO_ROOT/third-party/ao" > /dev/null
+  USE_CPP=0 "$PYBIN" setup.py develop
+  popd > /dev/null
```

Comment:
Given how these are stored in install_requirements.py, it feels a bit flaky. I think you can just refactor install_requirements.py so that you have torch_pin.py that defines TORCH_VERSION and NIGHTLY_VERSION. That would be cleaner.

Reply:
That comment was for torch

Reply:
Sure, add it in https://github.com/pytorch/executorch/pull/14631

Reply:
ok rebase on top of it and fix this?